porcupine/plugins/filemanager.py

```diff
@@ -358,12 +455,22 @@ def new_file_here(path: Path) -> None:
         get_tab_manager().event_generate("<<FileSystemChanged>>")
         get_tab_manager().open_file(name)
 
+def new_directory_here(path: Path) -> None:
+    # print('New Directory Created')
+    name = ask_directory_name(path, "", mode=DirectoryName.NEW)
+    if path.is_dir():
+        path=path.parent
+        name.mkdir()
```

Comment:
If I right-click a directory, choose "New directory here", and then click Cancel, I get:
```
Traceback (most recent call last):
  File "/usr/lib/python3.9/tkinter/__init__.py", line 1892, in __call__
    return self.func(*args)
  File "/home/akuli/porcu/porcupine/plugins/filemanager.py", line 463, in new_directory_here
    name.mkdir()
AttributeError: 'NoneType' object has no attribute 'mkdir'
```